source/virtualBuffers/gecko_ia2.py

```diff
@@ -355,19 +279,13 @@ def __contains__(self,obj):
 			or not winUser.isDescendantWindow(self.rootNVDAObject.windowHandle, obj.windowHandle)
 		):
 			return False
-		for accId in self._iterIdsToTryWithAccChild(obj):
-			if accId == self.rootID:
-				return True
-			try:
-				self.rootNVDAObject.IAccessibleObject.accChild(accId)
-				# The object is definitely a descendant of the document.
-				break
-			except COMError:
-				pass
-		else:
-			# The object is definitely not a descendant of the document.
+		accId = obj.IA2UniqueID
+		if accId == self.rootID:
+			return True
+		try:
+			self.rootNVDAObject.IAccessibleObject.accChild(accId)
+		except COMError:

```

Comment:
can you add logging here

Reply:
I don't think we want logging here. This exception will occur for any object that is not inside this document. Keep in mind that we potentially ask *every object* whether it is inside *every buffer*, so this will result in a massive amount of useless log spam. 99.9% of the time, this exception doesn't indicate a failure; it indicates that the object is not present in the document. In Pythonic terms, we might write this as:
`if id in document:`
but we're dealing with COM + IAccessible here, not pure Python.

Reply:
Thanks for the explanation

Reply:
I did tighten this up to make it clear which error codes aren't actually errors and to log errors we don't expect.